src/components/PokemonCollection.js

```diff
@@ -1,13 +1,25 @@
-import React from "react";
-import PokemonCard from "./PokemonCard";
-import { Card } from "semantic-ui-react";
+import React from "react"
+import PokemonCard from "./PokemonCard"
+import { Card } from "semantic-ui-react"
+
+function PokemonCollection({ pokemonList, searchedPokemon }) {
+  const displayPokemon = () => {
+    return pokemonList.map((pokemon) => {
+      return <PokemonCard key={pokemon.id} pokemon={pokemon} />
+    })
+  }
+
+  const displaySearchedPokemon = () => {
+    return searchedPokemon.map((pokemon) => {
+      return <PokemonCard key={pokemon.id} pokemon={pokemon} />
+    })
+  }
```

Comment:
If you do handle search here, which isn't preferable in my opinion, this is a refactor that you could consider since you have some duplication:
```suggestion
  const displayPokemon = () =>
    (searchedPokemon || pokemonList).map((pokemon) => (
      <PokemonCard key={pokemon.id} pokemon={pokemon} />
    ))
```